external-builds/pytorch/smoke-tests/run_smoke_tests.sh

```diff
@@ -3,6 +3,18 @@ set -xeuo pipefail
 
 echo 'Running inside the container'
 
+echo 'rocminfo' 
+rocminfo
+
+echo 'rocm-smi'
+rocm-smi
+#  Check for ROCm GPU availability
+echo 'Checking for ROCm-compatible GPU...'
+if ! rocminfo | grep -q 'Uuid: *GPU'; then
+  echo "ERROR: No ROCm-compatible GPU detected."
+  exit 1
+fi
```

Comment:
These tools do not exist on Windows and we should not be putting this much _test_ logic in a bash script.

We could run https://github.com/ROCm/TheRock/blob/main/tests/test_rocm_sanity.py as part of pytorch smoketests. That file already has test skipping per-platform already.

Reply:
Ohh Ok, mine was still in progress by the way. I did not know such a file exist -  will update the code